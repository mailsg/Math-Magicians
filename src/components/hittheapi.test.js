import React from 'react';
import renderer from 'react-test-renderer';
import HitTheApi from './hittheapi';

it('HitTheApi renders correctly', () => {
    const tree = renderer
        .create(<HitTheApi />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})