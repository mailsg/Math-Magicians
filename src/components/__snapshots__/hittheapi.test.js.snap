// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HitTheApi renders correctly 1`] = `
<div
  className="quotation"
>
  <p>
    Loading...
  </p>
</div>
`;
